portfolio/src/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
```

Comment:
It looks like there are two DataServlet.java files. I think this one can be removed.